src/Microsoft.OData.Core/UriParser/Binders/InBinder.cs

```diff
@@ -313,5 +326,30 @@ private static string NormalizeGuidCollectionItems(string bracketLiteralText)
 
             return "[" + String.Join(",", items) + "]";
         }
+
+        private static string NormalizeDateTimeCollectionItems(string bracketLiteralText)

```

Comment:
To ensure there are no bugs in this method, add a few tests with one parameter
e.g
`FilterClause filter = ParseFilter("Birthdate in (1950-01-02T06:15:00Z)",
                HardCodedTestModel.TestModel, HardCodedTestModel.GetPersonType());`

`FilterClause filter = ParseFilter("Birthdate in (null)",
                HardCodedTestModel.TestModel, HardCodedTestModel.GetPersonType());`

Reply:
I have added two tests with date/time collections containing single item or single null